FirebaseRemoteConfig/Sources/RCNPersonalization.m

```diff
@@ -37,16 +38,31 @@ - (void)logArmActive:(NSString *)key config:(NSDictionary *)config {
   }
 
   NSDictionary *metadata = ids[key];
-  if (!metadata || metadata[kPersonalizationId] == nil) {
+  NSString *personalizationId = metadata[kPersonalizationId];
+  if (!metadata || personalizationId == nil) {
     return;
   }
 
+  // This gets dispatched to a serial queue, so this is OK. But even if not, it'll just possibly log
+  // more.
+  if (self->_armsCache[key] == personalizationId) {
```

Comment:
does this mean we only want to log each arm once?

Reply:
We want to log at least once. We can do de-duping on the back-end.